artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/RemoteQueueBindingImpl.java

```diff
@@ -242,7 +243,10 @@ public synchronized void removeConsumer(final SimpleString filterString) throws
          }
       }
 
-      consumerCount--;
+      if (--consumerCount < 0) {
+         ActiveMQServerLogger.LOGGER.consumerCountError("Tried to decrement consumer count below 0: " + this);
```

Comment:
Is this new error necessary?


I have been trying to use more log.debug for things not really pertaining to the user.

Reply:
It is not strictly necessary no, I can remove it or add some debug logging